starboard/android/shared/player_components_factory.h

```diff
@@ -199,8 +199,6 @@ class PlayerComponentsFactory : public starboard::shared::starboard::player::
 
   const int kAudioSinkFramesAlignment = 256;
   const int kDefaultAudioSinkMinFramesPerAppend = 1024;
-  const int kDefaultAudioSinkMaxCachedFrames =
```

Comment:
`kDefaultAudioSinkMaxCachedFrames` is also defined in [player_components.cc](https://github.com/youtube/cobalt/blob/0f5cd1b0e15d428a994999ebdfd5f42c2bdf4971/starboard/shared/starboard/player/filter/player_components.cc#L43) but doesn't seem to be used - might be good to remove it in this change as well.

Reply:
Done